### PR TITLE
Add GTF getters for tx bytes size and address

### PR DESCRIPTION
**Pull request by Dentosal**
Closes https://github.com/FuelLabs/fuel-specs/issues/554

VM PR: https://github.com/FuelLabs/fuel-vm/pull/692

### CODE REVIEW

File: src/fuel-vm/instruction-set.md [hunk 1]
```diff
@@ -2473,6 +2473,8 @@ Get [fields from the transaction](../tx-format/transaction.md).
 | `GTF_SCRIPT_INPUT_AT_INDEX`               | `0x00B` | Memory address of `tx.inputs[$rB]`                                |
 | `GTF_SCRIPT_OUTPUT_AT_INDEX`              | `0x00C` | Memory address of `t.outputs[$rB]`                                |
 | `GTF_SCRIPT_WITNESS_AT_INDEX`             | `0x00D` | Memory address of `tx.witnesses[$rB]`                             |
+| `GTF_TX_START_ADDRESS`                    | `0x0A0` | Memory address of the raw transaction bytes                         |
```
Comment (luizstacio) [inline src/fuel-vm/instruction-set.md:2476]: Should we remove the address at the end, to follow other gtfs?

File: src/fuel-vm/instruction-set.md [hunk 2]
```diff
@@ -2473,6 +2473,8 @@ Get [fields from the transaction](../tx-format/transaction.md).
...
+| `GTF_TX_LENGTH`                           | `0x0A1` | Length of raw transaction types in memory                        |
```
Comment (xgreenx) [inline src/fuel-vm/instruction-set.md:2477]: Why do we want to put size and offset under `0x0A1` and `0x0A0`? Maybe it is better to put them at the end

File: src/fuel-vm/instruction-set.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Dentosal) [reply src/fuel-vm/instruction-set.md:2476]: deb35bca

File: src/fuel-vm/instruction-set.md (hunk 2 above)
Comment (Dentosal) [reply src/fuel-vm/instruction-set.md:2477]: No real reason. The location just seemed appropriate. I'll move them. deb35bca

**Review by xgreenx — APPROVED** [review]

**Pull request merged**